catalogd/internal/controllers/core/clustercatalog_controller_test.go

```diff
@@ -818,16 +818,10 @@ func TestPollingRequeue(t *testing.T) {
 		t.Run(name, func(t *testing.T) {
 			reconciler := &ClusterCatalogReconciler{
 				Client: nil,
-				Unpacker: &MockSource{result: &source.Result{
-					State: source.StateUnpacked,
-					FS:    &fstest.MapFS{},
-					ResolvedSource: &catalogdv1.ResolvedCatalogSource{
-						Image: &catalogdv1.ResolvedImageSource{
-							Ref: "my.org/someImage@someSHA256Digest",
-						},
-					},
-					LastSuccessfulPollAttempt: tc.lastPollTime,
```

Comment:
Hmm, this was not carried forward, but it didn't seem to cause tests to fail. Need to look more into this.

Reply:
Fixed up the test scenario to ensure the correct code path was actually being tested (and added another important test case to prove it!)